packages/jellyfish-api-core/__tests__/category/account/listBurnHistory.test.ts

```diff
@@ -0,0 +1,224 @@
+      expect(typeof burnHistory.blockHeight).toStrictEqual('number')
+      expect(typeof burnHistory.blockHash).toStrictEqual('string')
+      expect(typeof burnHistory.blockTime).toStrictEqual('number')
+      expect(typeof burnHistory.type).toStrictEqual('string')
+      expect(typeof burnHistory.txn).toStrictEqual('number')
+      expect(typeof burnHistory.txid).toStrictEqual('string')
+      expect(burnHistory.amounts.length).toBeGreaterThan(0)
+      expect(typeof burnHistory.amounts[0]).toStrictEqual('string')
+    }
+  })
+
+  it('first transaction should be masternode creation fee burn', async () => {
+    const history = await client.account.listBurnHistory()
+    expect(history[6].owner.slice(0, 16)).toStrictEqual('6a1a446654784301')
+    expect(history[6].type).toStrictEqual('CreateMasternode')
+    expect(history[6].txn).toStrictEqual(2)
+    expect(history[6].amounts.length).toStrictEqual(1)
+    expect(history[6].amounts[0]).toStrictEqual('1.00000000@DFI')
+  })
+
+  it('second transaction should be after burn token', async () => {
+    const history = await client.account.listBurnHistory()
+    expect(history[5].owner).toStrictEqual('6a19446654785404474f4c4404474f4c4408000000000000000007')
+    expect(history[5].type).toStrictEqual('CreateToken')
+    expect(history[5].txn).toStrictEqual(1)
+    expect(history[5].amounts.length).toStrictEqual(1)
+    expect(history[5].amounts[0]).toStrictEqual('1.00000000@DFI')
+  })
+
+  it('third transaction should be send token to burn address', async () => {
+    const history = await client.account.listBurnHistory()
+    expect(history[4].owner).toStrictEqual(burnAddress)
+    expect(history[4].type).toStrictEqual('AccountToAccount')
+    expect(history[4].amounts.length).toStrictEqual(1)
+    expect(history[4].amounts[0]).toStrictEqual('100.00000000@GOLD')
+  })
+
+  it('fourth transaction should be utxostoaccount burn', async () => {
+    const history = await client.account.listBurnHistory()
+    expect(history[3].owner).toStrictEqual(burnAddress)
+    expect(history[3].type).toStrictEqual('UtxosToAccount')
+    expect(typeof history[3].amounts).toStrictEqual('object')
+    expect(history[3].amounts.length).toStrictEqual(1)
+    expect(history[3].amounts[0]).toStrictEqual('1.00000000@DFI')
+  })
+
+  it('fifth transaction should be send to burn address with accountToAccount', async () => {
+    const history = await client.account.listBurnHistory()
+    expect(history[2].owner).toStrictEqual(burnAddress)
+    expect(history[2].type).toStrictEqual('AccountToAccount')
+    expect(history[2].amounts.length).toStrictEqual(1)
+    expect(history[2].amounts[0]).toStrictEqual('1.00000000@DFI')
+  })
+
+  it('sixth transaction should be send to burn address with accounttoutxos', async () => {
+    const history = await client.account.listBurnHistory()
+    expect(history[1].owner).toStrictEqual(burnAddress)
+    expect(history[1].type).toStrictEqual('None')
```

Comment:
I was thinking why this is `None`. Should be `AccountToUtxos`

Reply:
guess something wrong from the `ain`

Reply:
I thought so too yesterday